### PR TITLE
Automated PR for Multics

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/Multics.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!-- REVIEW -->
```
Comment (sam-ellis) [inline src/Multics.xml]: Remove this

File: src/Multics.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    </optional>
+    <body>
+      <p>Permission to use, copy, modify, and distribute these programs and their documentation for any purpose
+         and without fee is hereby granted,provided that the below copyright notice and historical background
+         appear in all copies and that both the copyright notice and historical background and this permission
+         notice appear in supporting documentation, and that the names of MIT, HIS, BULL or BULL HN not be used
+         in advertising or publicity pertaining to distribution of the programs without specific prior written
+         permission.</p>
+    </body>
+    <optional>
```
Comment (sam-ellis) [inline src/Multics.xml]: Consider adding \<copyright> tags.

File: src/Multics.xml (hunk 1 above)
Comment (bradleeedmondson) [inline src/Multics.xml]: thanks Sam, I can remove this

**Comment by bradleeedmondson** [comment]
also query whether that copyright text should be required, since it's required as part of the required acknowledgment

**Pull request merged**